Add new GSS functions with the fix for color functions (#821)
memgraph/docs: lab/style-script/gss-functions.md

- `List[any]`

### `Reduce(array, function, initialValue)`

> Returns a single value generated by reducing an array of values with a reduce function.
> A function has two parameters, `previous reduced value` and `current array value`. The
> third parameter is the initial value used for the first reduce iteration.

Reviewer: ```suggestion
The `Reduce()` function returns a single value generated by reducing an array of values.
The `function `parameter has two arguments, `previous reduced value` and `current array value`.
The `initalValue` parameter specifies the initial value used for the first reduce iteration. 
```
